docs/interfaces.md

```diff
@@ -379,3 +379,9 @@ Can manage timeservers directly separate from ``config core``.
 Can access the tpm device /dev/tpm0.
 
 * Auto-Connect: no
+
+### fwupd
+
+Allow fwupd to update UEFI capsule format firmware.
```

Comment:
@jdstrand Right, I agreed with Tim that we put all these things in a fwupd interface for now and split the UEFI bits out into a proper uefi-control interface at a later point.